src/slackNotification/slack_notification.py

```diff
@@ -14,8 +14,10 @@ def lambda_handler(event, context):
         message[constants.ERROR] = event['taskname'] + 'Error'
         message[constants.CAUSE] = event['status']
     elif 'Error' in event:
-        message[constants.ERROR] = event['Error']
-        message[constants.CAUSE] = event['Cause']
+        message['database id'] = event['databaseid']
+        message['snapshot id'] = event['snapshotid']
```

Comment:
1. can we move the initialisation of `result['database id']` and `result['snapshot id']` outside of the if else loop as these will be common. 

2. here too - the snapshot id will be of the format `event['Identifier'] + constants.SNAPSHOT_POSTFIX`